README.md

```diff
@@ -840,6 +840,23 @@ May also be configured in `lerna.json`:
 }
 ```
 
+#### --yarnWorkspaces
+
+Can be only configured in `lerna.json`:
+
+```js
+{
+  ...
+  "npmClient": "yarn",
+  "yarnWorkspaces": ["bootstrap"]
```

Comment:
I don't think this is an appropriate place to whitelist commands; that's an implementation detail _if_ workspaces are enabled at all. In this PR alone there's already an implicit `"ls"` member in this array, and that's not very clear. The `useWorkspaces` (the camelization of `--use-workspaces`) value should be a boolean.